src/components/FeatureViews/Scheduler/Scheduler.js

```diff
@@ -76,7 +76,7 @@ const Scheduler = () => {
     const [instructorFilter, setInstructorFilter] =
         useState(prevState.instructorFilter);
     const [timeShift, setTimeShift] = useState(prevState.timeShift || 0);
-    const [view, setView] = useState(prevState.view || "timeGridDay");
+    const timeView = props.location.state ? "timeGridDay" : prevState.view || "timeGridDay";    const [view, setView] = useState(timeView)
```

Comment:
This breaks currently - you should be accessing location via the `useLocation` hook from `react-router-dom`. Also, you should be checking for a specific part of the state (the `isDashboard` boolean), rather than whether or not there is a state. [Optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) may be useful.